tests/items/test_item_collection.py

```diff
@@ -57,3 +57,36 @@ def test_load_items_csv(domain: Domain, movie: Dict[str, Any]) -> None:
     for property in ["TITLE", "GENRE"]:
         assert item.get_property(property) == movie[property]
     assert item.get_property("KEYWORD") is None
+
+
+def test_get_possible_property_values(domain: Domain) -> None:
+    """Tests using slot with different types (str, list) and unknown slot."""
+    item_collection = ItemCollection()
+    mapping = {
+        "title": {"slot": "TITLE"},
+        "genres": {
+            "slot": "GENRE",
+            "multi-valued": True,
+            "delimiter": "|",
+        },
+    }
+
+    item_collection.load_items_csv(
+        ITEMS_CSV_FILE,
+        id_col="movieId",
+        domain=domain,
+        domain_mapping=mapping,
+    )
+
+    genres = item_collection.get_possible_property_values("GENRE")
```

Comment:
Also check for a value that's not supposed to be in a list of genres?

Reply:
Done.